js/views/pageNavVw.js

```diff
@@ -91,17 +96,48 @@ module.exports = Backbone.View.extend({
   },
 
   handleSocketMessage: function(response) {
-    var data = JSON.parse(response.data);
-    if(data.id == this.socketNotificationID){
-      console.log(data);
-    }
-  },
+    var data = JSON.parse(response.data),
+        username,
+        avatar,
+        n;
```

Comment:
Try to use verbose variable names. This is open source, and we want to make sure it's easy to understand for people that want to contribute.